samples/client/petstore/python-asyncio/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.3.1-SNAPSHOT
```

Comment:
Thank you for your PR. Generally it looks great; can you rebase?
These version file should not be changing.

Reply:
Sorry about that, fixed.